tests/ProxyManagerTest/ProxyGenerator/NullObjectGeneratorTest.php

```diff
@@ -80,7 +80,7 @@ public function testGeneratesValidCode(string $className)
         $proxyGenerated = $generatedClassName::staticProxyConstructor();
 
         foreach (Properties::fromReflectionClass($generatedReflection)->getPublicProperties() as $property) {
-            $this->assertNull($proxyGenerated->$property);
+            $this->assertNull($proxyGenerated->{$property->getName()});
```

Comment:
Similar issue to what I mentioned in my other comment. :) To illustrate, check out `master`, leave only `[BaseClass::class]` and `[ClassWithMixedProperties::class]` in the dataprovider, and add a `die` right before `$this->assertNull($proxyGenerated->$property);`.

That assertion only gets run if it's able to reflect any properties, which it doesn't because the original class isn't extended.